react/features/virtual-background/components/VirtualBackgroundPreview.tsx

```diff
@@ -1,29 +1,40 @@
-// @flow
-
+/* eslint-disable lines-around-comment */
 import Spinner from '@atlaskit/spinner';
 import { withStyles } from '@material-ui/core/styles';
 import React, { PureComponent } from 'react';
+import { WithTranslation } from 'react-i18next';
```

Comment:
maybe create a global.d.ts file so you can export some widely used types/interfaces globally without needing to import them on each file. In case you create a type/interface for eg styles, this would be usefull.

Reply:
Hmm, I have mixed feelings about this, but I'm fine either way. @saghul any thoughts?

Reply:
Not a fan TBH.